### PR TITLE
sql: fixing silent logic test failures when invalid testfile specified

### PR DESCRIPTION
**Pull request by RaduBerinde**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5315)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,6 +35,7 @@ TESTFLAGS    :=
 STRESSFLAGS  := -stderr -maxfails 1
```
Comment (tamird) [inline Makefile]: in the commit message:
> make stress should in the package directory

missed the word run
> When using `make test` the tests runs in the package directory. When using `make

mixing singular and plural in "tests runs"
> stress`, it ran in the current (cockroach) directory. If we are passing a path

"it" is singular
> via TESTFLAGS, the distinction is important. This change makes the stress
program run under the package directory.

"stress program" singular again

**Comment by tamird** [comment]
LGTM mod grammar in the second commit message

**Comment by RaduBerinde** [comment]
Fixed, thanks!

---







*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/5315#-:-KD0Ce6iEWZI1mgrMbSK:1154580981)*

**Comment by tamird** [comment]
Reviewed 1 of 1 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/5315)*

**Pull request merged**